docs/introduction/diffusion_intro/diffusion_intro.rst

```diff
@@ -0,0 +1,142 @@
+
+Mean Squared Displacement analysis
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+A common method for determining the diffusion coefficient, is to first compute the mean squared displacement (MSD) as a function of lag time for each track. 
+For purely diffusive motion, the first part of the MSD curve is linear, with the slope proportional to the diffusion coefficient. For 1D diffusion the MSD is given by: :math:`MSD = 2D\Delta t`.
+The diffusion coefficient can be obtained by fitting a linear function to the MSD curve, as illustrated in the figure below (simulated data):
+
+.. image:: figures/MSD_analysis.png
+
+Note that for the MSD curve, the 'lag time' (aka 'time interval') is given on the x-axis, rather than 'time'. 
+To compute the MSD for a certain lag time, you scan over a track, one data point at a time and compute the displacement squared 
+from that time until that time + the lag time. So each point on the MSD curve, is an average of displacements for a specific lag time  (see page ... for the equation of MSD). 
+
+The largest possible lag time is the duration of a track and the shortest possible lag is the time resolution, the line time of the kymograph. 
+
+Note that the image above does not show the complete MSD curve, but only up to a lag time of 5 seconds. 
+The full MSD curve looks as follows:
+
+.. image:: figures/MSD_full.png
+
+As you can see, the full MSD curve becomes less and less linear for larger lags times. The funky shape of the MSD curve for large lag times is caused by:
+
+#. The MSD at shorter time lags is averaged over more lags and therefore more accurate
+
+#. Consecutive data points in the MSD are highly correlated
+
+The MSD curve for larger lags is not suitable for a linear fit. An important step in MSD analysis, is therefore to determine which number of lags is optimal for a linear fit and obtaining the diffusion coefficient.
+Taking too many lags, may result in an MSD curve that is not fully linear, while taking too little lags may not give enough data points for a good estimate of the diffusion coefficient.
+There are various methods for determining the optimal number of lags. The method recommended in Pylake automatically determines the optimal number of lags for you :cite:`michalet2012optimal`(named 'OLS' in Pylake). 
+
+Covariance Based Estimator
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The Covariance Based Estimator (:ref:`'CVE' <diffusion_cve>`) in Pylake is a method for determining the diffusion coefficient that does not rely on the MSD :cite:`vestergaard2014optimal,vestergaard2016optimizing`. 
+(CITE STIGLER) This method directly computes the diffusion coefficient from the displacements.
+
+Comparing CVE and MSD analysis
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Since there is no fitting involved, CVE is fast and unbiased (unbiased means that the more data you get, the closer the average gets to the real diffusion constant). 
```

Comment:
I really like that you're adding some more user-friendly documentation.

Sorry about the drive-by comment, I know you said it wasn't ready for review yet, but I noticed this and wanted to leave a quick comment:

- This sentence makes it seem like fitting causes the bias but it doesn't. The bias in the MSD mostly comes from the fact that it is very difficult to design an unbiased estimator based on that quantity (because of the non-linearity applied when estimating it and the correlation structure involved when you estimate it with time averaging). I would also argue that fitting doesn't necessarily make things slow (Since the regular MSD is a very basic linear fit, which is virtually instantaneous. It's the calculation of the MSD itself that costs time because of the calculation of all the measurement pairs).

- The second thing is that the definition of bias presented here is not the statistical definition of bias. What you describe sounds more like consistency (that as `N->inf`, `theta_est -> theta`). I'm honestly not sure whether the MSD is a consistent estimator either, but it might be.

Statistical bias is a property of finite samples. It means that if you were to make a distribution of estimates based on finite samples, the expectation (average) of that distribution would not be the true population value. It is a property for a specific `N`.

A good example of a biased but consistent estimator is the sample variance estimator. This is when you estimate the variance plugging in the mean from the same sample and you don't take into account the Bessel correction. For an infinite number of samples, it will converge to the true value, but if you take the average of many samples computed like this, the one with the Bessel correction will produce the correct result, while the one without will reveal its bias.

A good example of an unbiased but consistent estimator of the population mean would be just taking the first element of the sample. It's clearly unbiased (the average over many of these estimates based on samples from the population would result in the correct estimate), but it will never converge as the sample size for a single estimate goes to infinity.

I would just mention that the CVE is fast and unbiased and leave it at that. I feel bias, consistency and precision could almost be their own theory pages.